torchgeo/datasets/vaihingen.py

```diff
@@ -0,0 +1,400 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Vaihingen dataset."""
+
+import os
+from typing import Any, Callable, Dict, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytorch_lightning as pl
+import torch
+from matplotlib.figure import Figure
+from PIL import Image
+from torch import Tensor
+from torch.utils.data import DataLoader
+from torchvision.transforms import Compose
+
+from ..datasets.utils import dataset_split, draw_semantic_segmentation_masks
+from .geo import VisionDataset
+from .utils import check_integrity, extract_archive, rgb_to_mask
+
+
+class Vaihingen2D(VisionDataset):
+    """Vaihingen 2D Semantic Segmentation dataset.
+
+    The `Vaihingen <https://www2.isprs.org/commissions/comm2/wg4/benchmark/2d-sem-label-vaihingen/>`_
+    dataset is a dataset urban semantic segmentation used in the 2D Semantic Labeling
+    Contest - Vaihingen. This dataset uses the "ISPRS_semantic_labeling_Vaihingen.zip"
```

Comment:
Same comments as other PR, this sentence does not make sense

Reply:
It should be "a dataset for urban semantic..."